stoplight.js

```diff
@@ -0,0 +1,12 @@
+$("#red").toggle(function() {
+  $("#red").addClass(".active #red")
+}, function() {
+  $("#red").removeClass(".active #red")
+}
+})
+$("#yellow").toggle(function() {
+  $("#yello").addClass(".active #yellow")
+})
+$("#green").toggle(function() {
+  $("#green").addClass(".active #green")
+})
```

Comment:
Make sure you're testing your code as you write it. Look into how the `toggle` method works specifically.